### ISSUE TITLE
Support https://adhearsion.com/

### ISSUE THREAD

**Issue by sfgeorge** [adhearsion/adhearsion-website#39]
I think we need to cover the following bases:
1. Get https://adhearsion.com/ to work
2. Redirect all http traffic to https

Having a site on http:// has a negative impact on security and visitor confidence:

![image](https://user-images.githubusercontent.com/208529/42582669-5fc742aa-84f5-11e8-99f4-04dfbbed5449.png)

Additionally, it has a negative impact on SEO for those who are searching for a great platform for controlling communications.

There are a few good, free options for free SSL these days, including [Let's Encrypt](https://letsencrypt.org/) and CloudFlare's Universal SSL.

**Comment by chewi** [comment]
The ageing gems are also a concern. However, most of the dynamic elements seem redundant now. Would it make more sense to turn it into a static site? Probably cheaper too.

**Comment by sfgeorge** [comment]
Good point, that would reduce surface area of risk.

**Comment by benlangfeld** [comment]
If you used middleman, you could render the existing templates at compile-time, and just host from S3 or similar.

**Comment by chewi** [comment]
@benlangfeld, exactly what I was thinking as I've used Middleman before. Would GitHub Pages even be an option?

**Comment by benlangfeld** [comment]
Yes, looks like it. If someone can get middleman working, just let me know if there’s anything you need me to do with configuration/DNS/etc

**Comment by sfgeorge** [comment]
Awesome.  If we end up doing middleman -> github pages, I think I found the necessary deets for SSL of custom github-io domains here:

[GitHub Blog: Custom domains on GitHub Pages gain support for HTTPS](https://blog.github.com/2018-05-01-github-pages-custom-domains-https/#configuring-your-domain)

Basically may just require a CNAME 💪

**Comment by sfgeorge** [comment]
Sorry, please ignore my CNAME suggestion.  @adhearsion.com has active MX recordings, and using a CNAME will break mail functionality for the domain.  [as suggested here on S.O.](https://webmasters.stackexchange.com/a/105971/89695), a good approach is to use A, ALIAS or ANAME - but not a CNAME.

[GitHub Help: Setting up an apex domain
](https://help.github.com/articles/setting-up-an-apex-domain/)
